packages/@webex/webex-core/test/integration/spec/services-v2/services-v2.ts

```
@@ -0,0 +1,1015 @@
+      before('collect remote catalogs', () =>
+        Promise.all([
+          services._fetchNewServiceHostmap(),
+          services._fetchNewServiceHostmap({
+            from: 'limited',
+            query: {userId: webexUser.id},
+          }),
+        ]).then(([fRHM, lRHM]) => {
+          fullRemoteHM = fRHM;
+          limitedRemoteHM = lRHM;
+        })
+      );
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow before duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)